src/services/azblob/backend.rs

```diff
@@ -39,6 +40,15 @@ use crate::*;
 
 const X_MS_BLOB_TYPE: &str = "x-ms-blob-type";
 
+/// Known regex pattern Azure Storage Blob services resource URI syntax.
```

Comment:
I don't think introducing regex is necessary for such a simple task. Additionally, it's not our responsibility to validate the account name; that falls on the user.

Instead, how about we just maintain a list of valid suffixes?

Reply:
Done.